apps/coscrad-frontend/src/components/resources/spatial-features/thumbnail-presenters/spatial-feature-detail.thumbnail.presenter.tsx

```diff
@@ -5,24 +5,19 @@ import {
     ResourceType,
 } from '@coscrad/api-interfaces';
 import { isNullOrUndefined } from '@coscrad/validation-constraints';
-import { ExpandLess as ExpandLessIcon, ExpandMore as ExpandMoreIcon } from '@mui/icons-material';
+import { Card, CardContent } from '@mui/material';
 import {
-    Card,
-    CardActionArea,
-    CardActions,
-    CardContent,
-    CardHeader,
-    Collapse,
-    Divider,
-} from '@mui/material';
+    FloatSpacerDiv,
+    SinglePropertyPresenter,
+} from 'apps/coscrad-frontend/src/utils/generic-components';
 import { useState } from 'react';
-import { Link } from 'react-router-dom';
 import { routes } from '../../../../app/routes/routes';
 import { FunctionalComponent } from '../../../../utils/types/functional-component';
+import { ResourceNavLink } from '../../shared/resource-nav-link';
 import { LineTextPresenter } from './line-text-presenter';
 import { PointTextPresenter } from './point-text-presenter';
 import { PolygonTextPresenter } from './polygon-text-presenter';
-import { toGeoJSON } from './to-geo-json';
```

Comment:
Why was this removed? We should definitely allow the user to get proper GeoJSON to copy to the clipboard.